src/rf/apps/core/models.py

```diff
@@ -230,10 +243,6 @@ class LayerImage(Model):
         max_length=255,
         help_text='Filename of original file',
     )
-    s3_uuid = UUIDField(
```

Comment:
I'd be more in favor of leaving the uuid field and removing the uri field since that could be calculated from the uuid if we stored the file extension type somewhere.
`uri = user_id + '-' + uuid + '-' + '.' + file_extension`